RecoParticleFlow/PFProducer/python/particleFlowSimParticle_cfi.py

```diff
@@ -35,3 +30,7 @@
     #retrieving fastSim SimHits                                     
     fastSimProducer = cms.untracked.InputTag('fastSimProducer','EcalHitsEB')
 )
+
+# Custom setting
+particleFlowSimParticle.ParticleFilter.chargedPtMin = cms.double(0.0)
+particleFlowSimParticle.ParticleFilter.EMin = cms.double(0.0)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.